Jenkinsfile

```diff
@@ -44,7 +44,9 @@ def smokeTest(gitUtils, bazel, utils) {
   goBuildNode(gitUtils, 'istio.io/istio') {
     bazel.updateBazelRc()
     utils.initTestingCluster()
-    def e2eArgs = "--logs_bucket_path ${gitUtils.logsPath()} "
+    def logHost = 'stackdriver'
+    def projID = 'istio-testing'
+    def e2eArgs = "--logs_bucket_path ${gitUtils.logsPath()} --log_provider=${logHost} --project_id=${projID} "
```

Comment:
Take a look at line 23, your change only affects SMOKE_TEST for other repos. While here, we are running PRESUBMIT at line 20.